src/analytics/src/main/scala/osmesa/analytics/oneoffs/ChangesetStatsCreator.scala

```diff
@@ -27,6 +28,16 @@ object ChangesetStatsCreator
           Opts
             .option[String]("changesets", help = "Location of the Changesets ORC file to process.")
 
+      val changesetBaseOpt =
+        Opts
+          .option[URI](
+          "changeset-stream",
+          short = "c",
+          metavar = "uri",
+          help = "HTTP Location of replication changesets"
+        )
+        .validate("Changeset source must have trailing '/'") { _.getPath.endsWith("/") }
```

Comment:
Can we drop this requirement and update URIs to match within `ChangesetSource`?

This can also have a default value of `https://planet.openstreetmap.org/replication/changesets/` (which should really be a default value in `ChangesetSource`, as it gets duplicated all over right now.

Reply:
Noted in geotrellis/vectorpipe#111.